conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -72,9 +59,12 @@ conda install --yes --quiet conda-forge-build-setup
 {% if build_setup -%}
 {{ build_setup }}{% endif -%}
 
-conda build /recipe_root --quiet || exit 1
+# testing purposes: get conda-build from defaults
+conda install -n root --quiet --yes -c defaults conda-build
```

Comment:
Remove this line before merging (version constraint should come from conda-forge-build-setup)